app/Actions/Database/StartDatabaseProxy.php

```
@@ -42,6 +44,12 @@ public function handle(StandaloneRedis|StandalonePostgresql|StandaloneMongodb|St
             'standalone-mongodb' => 27017,
             default => throw new \Exception("Unsupported database type: $databaseType"),
         };
+        if ($isSSLEnabled) {
+            $internalPort = match ($databaseType) {
+                'standalone-redis', 'standalone-keydb', 'standalone-dragonfly' => 6380,
+                default => throw new \Exception("Unsupported database type: $databaseType"),
```

PHPMD (2.15.0)

Missing class import via use statement (line '50', column '38'). (Clean Code Rules)

(MissingImport)